projects/ngrx.io/src/styles/2-modules/details/_details-theme.scss

```diff
@@ -0,0 +1,12 @@
+@use '../../constants';
+@use '@angular/material' as mat;
+
+@mixin theme($theme) {
+  details {
+    box-shadow: 0 1px 4px 0 rgba(constants.$black, 0.37);
+  }
+
+  summary {
+    color: constants.$black;
```

Comment:
This will need to be updated, as it isn't very readable with the dark theme

For an example see https://deploy-preview-4054--ngrx-io.netlify.app/api/store/Store

![image](https://github.com/ngrx/platform/assets/28659384/9d1a1973-506d-4c59-95b8-9bc424fcd044)

Reply:
Fine, will take care of it tomorrow :)